app/controllers/users_controller.rb

```diff
@@ -0,0 +1,39 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def show
+    @users = User.find(params[:id])
+  end
+
+  def new
+    @users = User.new
+  end
+
+  def create
+    @users = User.new(user_params)
+
+    if @users.save
+      redirect_to @users
+    else
+      render 'new'
+    end
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to @user
+    else
+      render 'edit'
+    end
+  end
```

Comment:
- [ ] Please from the project requirement, you should NOT use the `rails generate scaffold` command to generate your controllers and your views. Remember that this project is to show you how to set up controllers by yourselves so make sure that you set up everything by yourself :smile: